core/gui/src/app/dashboard/service/user/download/download.service.ts

```diff
@@ -71,13 +110,57 @@ export class DownloadService {
     this.notificationService.info(startMessage);
     return retrieveFunction().pipe(
       tap(blob => {
-        this.fileSaverService.saveAs(blob, fileName);
-        this.notificationService.info(successMessage);
+        this.saveFile({ blob, fileName });
+        this.notificationService.success(successMessage);
       }),
       catchError((error: unknown) => {
         this.notificationService.error(errorMessage);
         return throwError(() => error);
       })
     );
   }
+
+  private saveFile({ blob, fileName }: DownloadableItem): void {
+    this.fileSaverService.saveAs(blob, fileName);
+  }
+
+  public downloadOperatorsResult(
```

Comment:
move this method up, next to those public methods

Reply:
Resolved!